lib/pure/httpclient.nim

```diff
@@ -297,6 +297,16 @@ proc newMultipartData*: MultipartData =
   ## Constructs a new ``MultipartData`` object.
   MultipartData(content: @[])
 
+
+proc `$`*(data: MultipartData): string =
+  ## convert MultipartData to string so it's human readable when echo
+  ## see https://github.com/nim-lang/Nim/issues/11863
+  for pos, item in data.content:
+    result &= "------------------------------  " 
+    result &= pos 
+    result &= "  ------------------------------\n"
+    result &= item[32 .. item.high]
```

Comment:
Where is this 32 coming from?

Reply:
32 is for strip out "Content-Disposition: form-data; " at the beginning.
It's for reduce visual noise
for full message will like
-----------------------------735323031399963166993862150
Content-Disposition: form-data; name="file3"; filename="binary"
Content-Type: application/octet-stream

sample from [here](https://stackoverflow.com/questions/4526273/what-does-enctype-multipart-form-data-mean)